content/communities/documenting-your-project-with-wikis/editing-wiki-content.md

```diff
@@ -45,7 +45,7 @@ Wikis can display PNG, JPEG, and GIF images.
 
 You can link to an image in a repository on {% data variables.product.github %} by copying the URL in your browser and using that as the path to the image. For example, embedding an image in your wiki using Markdown might look like this:
 
-    [[https://github.com/USERNAME/REPOSITORY/blob/main/img/octocat.png|alt=octocat]]
+    [[git@github.com:USERNAME/REPOSITORY/blob/main/img/octocat.png|alt=octocat]]
```

Comment:
Hi @FredrIQ, and thanks for your submission

Did you suggest editing the right page? It looks like you may have meant to suggest edits to https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages?

Additionally, I've just tested and I'm able to push changes to a wiki using HTTPS, so I'm not sure the docs are incorrect here. Is it possible something else has happened with your configuration? If so, this wouldn't be the best place to get support, but I'm happy to leave this open for now in case there is in fact a problem with the docs I'm not seeing